src/components/Garden/Metrics.js

```diff
@@ -29,68 +29,80 @@ const Metrics = React.memo(function Metrics({
           justify-content: space-around;
         `}
       >
-        <div
-          css={`
-            display: flex;
-            align-items: center;
-            margin-bottom: ${(compactMode ? 2 : 0) * GU}px;
-          `}
-        >
-          <img
-            src={token.logo || defaultTokenLogo}
-            height="60"
-            width="60"
-            alt=""
-            onClick={onExecuteIssuance}
+        {layoutName !== 'medium' && (
+          <div
             css={`
-              margin-right: ${4 * GU}px;
-              cursor: pointer;
+              display: flex;
+              align-items: center;
             `}
-          />
-          <TokenPrice currency={currency} token={token} />
-        </div>
+          >
+            <img
+              src={token.logo || defaultTokenLogo}
+              height="60"
+              width="60"
+              alt=""
+              onClick={onExecuteIssuance}
+              css={`
+                margin-right: ${4 * GU}px;
+                cursor: pointer;
+              `}
+            />
+            <TokenPrice currency={currency} token={token} />
+          </div>
+        )}
         <div>
           <TokenBalance
             label="Common Pool"
             value={commonPool}
             token={token}
             currency={currency}
+            helptip="common-pool"
           />
         </div>
         <div>
           <TokenBalance
-            label="Token Supply"
+            label="Total Supply"
```

Comment:
What do you think of check the Garden type, and display the `Wrapped Supply` for BYOT Gardens instead? Or maybe both? Given that CV calculations will be made considering the Wrapped Supply.

Reply:
I don't think wrapped is especially clear tbh

Reply:
but you raise a good point

Reply:
Let's think about how to best make this distinction

Reply:
Yes, good point. Maybe we could add the token symbol in parenthesis for BYOT gardens?

Reply:
Opened an issue for this https://github.com/1Hive/gardens/issues/323

Going ahead and merging